Components/explore.js

```diff
@@ -1,7 +1,27 @@
 import styles from '/styles/Products.module.css'
```

Comment:
Hi please remove the css modules and use styled component instead its not good idea to use both in same time

Reply:
done, removed the module.css on the parts i worked and will remove the remaining modules when i worked on that specific part.
also, moved the constant texts to their specific file